### PR TITLE
fix crashes when transformations are not passed in expected format

### PR DESCRIPTION
**Pull request by mszulik**

### CODE REVIEW

File: tests/Unit/ImageTest.php [hunk 1]
```diff
@@ -75,4 +79,141 @@ public function ensureUnprocessedFilesAreNotAvailable(Media $media)
...
+                'requestedTransformations' => '+',
+                'expectedException' => InvalidTransformationFormatException::class,
+            ],
+
+            'invalidFormatMissingMinus' => [
+                'requestedTransformations' => 'abc+q-50',
+                'expectedException' => InvalidTransformationFormatException::class,
+            ],
+
+            // Covers cases where the Transformation is missing.
```
Comment (gael-connan-cybex) [inline tests/Unit/ImageTest.php:194]: why

Comment (mszulik) [reply tests/Unit/ImageTest.php:194]: what

Comment (mszulik) [reply tests/Unit/ImageTest.php]: removed.

File: tests/Unit/ImageTest.php [hunk 2]
```diff
@@ -75,4 +79,216 @@ public function ensureUnprocessedFilesAreNotAvailable(Media $media)
...
+     */
+    public static function provideTransformationStrings(): array
+    {
+        return [
+            'valid_Quality' => [
+                'input' => 'q-50',
+                'expectedException' => null,
+                'expectedArray' => [
+                    'q' => 50,
+                ]
```
Comment (gael-connan-cybex) [inline tests/Unit/ImageTest.php:96]:
```suggestion
                ],
```

Comment (mszulik) [reply tests/Unit/ImageTest.php:96]: This is not done automatically by formatting, so I won't apply it.

**Review by gael-connan-cybex — APPROVED** [review]
tested

**Pull request merged**